src/libraries/Native/Unix/System.Security.Cryptography.Native/pal_pkcs12.c

```diff
@@ -1,67 +0,0 @@
-// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-
-#include "pal_pkcs12.h"
-
-PKCS12* CryptoNative_DecodePkcs12(const uint8_t* buf, int32_t len)
-{
-    if (!buf || !len)
-    {
-        return NULL;
-    }
-
-    return d2i_PKCS12(NULL, &buf, len);
-}
-
-PKCS12* CryptoNative_DecodePkcs12FromBio(BIO* bio)
-{
-    return d2i_PKCS12_bio(bio, NULL);
```

Comment:
Bonus points: For each function called from this shim, see if it can be removed from opensslshim.h (maybe we can also drop the include for openssl/pkcs12.h?)

Reply:
Good point on the shims. 

> maybe we can also drop the include for openssl/pkcs12.h

_Close_. We still need `d2i_PKCS12_fp`, `PKCS12_parse`, and `PKCS12_free` in X509:

https://github.com/dotnet/runtime/blob/e447e81bba126e78194e8fb66326579c01220116/src/libraries/Native/Unix/System.Security.Cryptography.Native/pal_x509.c#L424-L431